.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+<!-- Summary of what this PR introduces and possibly why -->
+
+## Changes
+
+<!-- List of relevant changes introduced -->
+
+## How Has This Been Tested?
```

Comment:
I'd argue it is a bit more expressive; the first time I read our "Testing" section I thought it meant "that's how you can test it" instead of "that's how I already tested it"

But I'm not in love with my own solution, I'm open to change it to Testing if you think it'd be better